### ISSUE TITLE
ciuploadtool mistakenly recreates tagged release

### ISSUE THREAD

**Issue by d1vanov** [d1vanov/ciuploadtool#5]
[Example](https://api.travis-ci.org/v3/job/451354273/log.txt) from whoozle/android-file-transfer-linux#205:
```
Running on Travis CI
Commit:  3475944e4d74e637f3676c4c571c482d398d7545
Found existing release but its commit SHA doesn't match the current one: 3475944e4d74e637f3676c4c571c482d398d7545 vs master
Deleting the existing release to recreate it with the current commit SHA 3475944e4d74e637f3676c4c571c482d398d7545
Creating new release
Created new release
Trying to upload file: ./Android_File_Transfer-3475944-x86_64.AppImage
Trying to upload file: ./Android_File_Transfer-3475944-x86_64.AppImage.zsync
```
It looks like ciuploadtool confused what was branch name with commit SHA, probably it is somehow related to `TRAVIS_COMMIT` environment variable's behaviour, need to investigate.

**Comment by d1vanov** [comment]
According to [this](https://developer.github.com/v3/repos/releases/#get-the-latest-release), GitHub API returns branch name in `target_commitish` field:

> target_commitish - string - Specifies the commitish value that determines where the Git tag is created from. Can be any branch or commit SHA. Unused if the Git tag already exists. Default: the repository's default branch (usually master).

Damn. Need to find some way around this...

**Comment by d1vanov** [comment]
Should be fixed after commit [121edec](https://github.com/d1vanov/ciuploadtool/commit/121edec31646153939400fb0e49a56116128b542) in branch [master](https://github.com/d1vanov/ciuploadtool/commit/master).
